### PR TITLE
added documentation of platform maintenance

### PR DESCRIPTION
**Pull request by sttomm**
I also did a pdf export, which is not looking that nice in teh end, but perhaps it helps the reviewers to get an insight on how it will roughly look like in the end

[meshPlatforms.pdf](https://github.com/meshcloud/meshcloud-docs/files/8222037/meshPlatforms.pdf)

### CODE REVIEW

File: docs/administration.platforms.md [hunk 1]
```diff
@@ -1,37 +1,70 @@
 ---
 id: administration.platforms
-title: Platform Operation
+title: meshPlatforms
 ---
 
-Operators can perform various maintainance operations for their platforms. For maintenance of Landing Zones, please see
-[Landing Zone Administration](meshcloud.landing-zones.md).
+A key task for Platform Operators is integrating your Cloud Platforms as meshPlatforms into meshStack. This enables
```
Comment (Jelledb) [inline docs/administration.platforms.md:6]: "as meshPlatforms" -> I would just add a short explanation on what that means. I could imagine first time users being a bit confused by the terminology.

File: docs/administration.platforms.md [hunk 2]
```diff
@@ -1,37 +1,70 @@
...
+tenant creation, replication and metering of your Cloud Platforms. The maintenance includes
```
Comment (Jelledb) [inline docs/administration.platforms.md:7]:
```suggestion
the core functionality of meshStack such as tenant creation, replication, and metering of your Cloud Platforms. The maintenance includes
```

File: docs/administration.platforms.md [hunk 3]
```diff
@@ -1,37 +1,70 @@
...
+general information about the Cloud Platform as well as structuring and configuring them.
```
Comment (Jelledb) [inline docs/administration.platforms.md:8]: I don't really get this sentence: "The maintenance [...] configuring them". What are you trying to convey here?

File: docs/administration.platforms.md [hunk 4]
```diff
@@ -1,37 +1,70 @@
...
 
-## Platform Notifications
+A central part of it also includes the application of security and governance which is covered in the
```
Comment (Jelledb) [inline docs/administration.platforms.md:10]: The two points you mention about landing zones and tenants would prob. go better in a bullet list. something like:

```
Besides integrating your cloud platform, the following functionality is also highly useful:
* Manage and create landing zones in the [landing zone administration screen](./linkto)
* View all your existing cloud tenants in the [tenant list](./linkto)
```
I think this also bridges better to your point of this functionality becoming a part of the platform control plane in future.

File: docs/administration.platforms.md [hunk 5]
```diff
@@ -1,37 +1,70 @@
...
+
+You can integrate new meshPlatforms in self-service via a wizard in meshStack.
+
+![Platform Creation](assets/platform_maintenance/platform-creation.png)
+
+The following platform types are available for selection.
+
+![Platform Types](assets/platform_maintenance/platform-types.png)
+
+You have to assign a meshLocation to your meshPlatform and can provide additional provider information to the meshLocation if you
```
Comment (Jelledb) [inline docs/administration.platforms.md:29]: I would recommend to add some best practices for the meshLocation. I could imagine that users do not really know what to do with them initially. Something like:

The meshLocation is useful for grouping together multiple platforms. This is especially helpful for private cloud platforms that have an actual physical location. For public cloud platform, we recommend to name them according to the provider as public cloud platforms are not really tied to a physical location. E.g. name the meshLocation for Azure simply "Azure".

File: docs/administration.platforms.md [hunk 6]
```diff
@@ -1,37 +1,70 @@
...
+would like to. This is the same information you can provide in the [Settings](#settings) tab of the Platform Control Plane.
```
Comment (Jelledb) [inline docs/administration.platforms.md:30]:
```suggestion
would like to. You can always change this information later in the [Settings](#settings) tab of the Platform Control Plane.
```

File: docs/administration.platforms.md [hunk 7]
```diff
@@ -1,37 +1,70 @@
...
+
+Additionally you can directly [restrict](#restrict-platform-access) your meshPlatform to certain meshCustomers to e.g. first test a new meshPlatform integration
+with a meshCustomer you have access to. This can prevent confusion of other users who may see the meshPlatform otherwise
```
Comment (Jelledb) [inline docs/administration.platforms.md:33]:
```suggestion
with a private meshCustomer that only you have access to. This can prevent confusion of other users who may see the meshPlatform otherwise
```

File: docs/administration.platforms.md [hunk 8]
```diff
@@ -1,37 +1,70 @@
...
+
+### Financials
+
+In this tab you can get an overview about which kind of resources are being used in your Cloud Platform.
+
+### Settings
 
-## Restrict Platform Access
+This tab is mainly responsible for managing how your meshPlatform is shown and used inside meshStack. You can provide general information
+about your Platform like a display name or the Web Console Url meshStack will link to when the user wants to actually access the cloud platform.
```
Comment (Jelledb) [inline docs/administration.platforms.md:51]: Maybe show a tip or smth: "For public cloud platforms you can leave the Web Console Url as is".

File: docs/administration.platforms.md [hunk 9]
```diff
@@ -1,37 +1,70 @@
...
 
-Access to a specific platform can be restricted in the Administration Area via "Platforms" -> "Platform Instances". This feature is helpful
+You can also group your meshPlatforms into meshLocations to e.g. combine multiple private cloud platforms that are running in the same data center
```
Comment (Jelledb) [inline docs/administration.platforms.md:52]: ah this is the point that I made before. Maybe you can move it up?

File: docs/administration.platforms.md [hunk 10]
```diff
@@ -1,37 +1,70 @@
...
+into one meshLocation. Or you are using e.g. multiple Azure Tenants for your different stages and want to group them into an Azure meshLocation.
+
+Additionally you can provide some Provider information of your meshLocation. You can describe how you operate your Cloud Platforms that belong to a meshLocation,
+you can provide a link to a Wiki to provide further, more detailed information about your meshPlatforms within a meshLocation.
+
+#### Restrict Platform Access
+
+Access to a specific platform can be restricted via the "Restrictions" tab. This feature is helpful
```
Comment (Jelledb) [inline docs/administration.platforms.md:60]: This was already mentioned before. I would suggest to structure the document in a way that you won't have to repeat yourself for the creation and updating part of things.

File: docs/administration.platforms.md [hunk 11]
```diff
@@ -41,3 +74,28 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 > Currently only OpenShift is supported for Quota Management
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
+
+### Platform Deactivation
+
+Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+
+Deactivation will make the meshPlatform not available anymore to new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. This will change in future. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform tenants when a meshPlatform is deactivated.
```
Comment (Jelledb) [inline docs/administration.platforms.md:82]: "This will change in future". This sounds a bit vague, can we by any chance already provide more info on the topic or even get rid of this sentence altogether?

File: docs/administration.platforms.md [hunk 12]
```diff
@@ -41,3 +74,28 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
+
+- **Platforms**: Select the platforms relevant to your notification so they can be shown on the affected platform's dashboards. Users
+  can also [subscribe](meshcloud.profile.md#profile) to notifications by platform and [subscribed users](./meshcloud.profile.md#platform-notification-subscriptions) will receive new notifications via email.
+- **Message**: The message that will be shown to all users. You can enter valid HTML in this field (except for the `<p>` tag). This is useful for e.g. supplying links to support pages.
+- **Severity**: The type of message (info, warning, critical).
+  - Info: for new features, upcoming services updates and general information that is not expected to disrupt platform operation
+  - Warning: best used to inform about partial platform outages or problems that may occur for some users
+  - Critical: appropriate in case of complete platform failures or impending platform downtimes
+- **Show From**: The earliest time the message will be shown. You can use this to create messages that will only be shown on at a later date.
+- **Show Until**: After this time the message will no longer be displayed.
```
Comment (Jelledb) [inline docs/administration.platforms.md:99]: For these two values there is also behavior when leaving them empty right? Maybe describe that too.

**Review by Jelledb — CHANGES_REQUESTED** [review]
One overall remark:

It feels very double that we have both `User Docs > Administration > meshPlatforms` & `Operator Docs > meshPlatforms`. Most of it is even duplicated information so I would propose getting rid of either of the two.

File: docs/administration.platforms.md (hunk 5 above)
Comment (j0g3sc) [inline docs/administration.platforms.md:29]: I would rephrase the part

> You have to assign a meshLocation to your meshPlatform and can provide additional provider information [...]

probably into two sentences, because one is a requirement and one is an option.

File: docs/administration.platforms.md [hunk 13]
```diff
@@ -1,37 +1,70 @@
...
+The following platform types are available for selection.
+
+![Platform Types](assets/platform_maintenance/platform-types.png)
+
+You have to assign a meshLocation to your meshPlatform and can provide additional provider information to the meshLocation if you
+would like to. This is the same information you can provide in the [Settings](#settings) tab of the Platform Control Plane.
+
+Additionally you can directly [restrict](#restrict-platform-access) your meshPlatform to certain meshCustomers to e.g. first test a new meshPlatform integration
+with a meshCustomer you have access to. This can prevent confusion of other users who may see the meshPlatform otherwise
+and would try to use it eventhough it is not working yet. Or you just have a dedicated Cloud Platform that shall only be available
```
Comment (j0g3sc) [inline docs/administration.platforms.md:34]:
```suggestion
and would try to use it eventhough it is not working yet. Another use case is to have a dedicated Cloud Platform that shall only be available
```

File: docs/administration.platforms.md [hunk 14]
```diff
@@ -1,37 +1,70 @@
...
+to certain meshCustomers in general.
+
+## Platform Control Plane
+
+The Platform Control Plane is the central entry point for any tasks or insights you want to do or get about your meshPlatform.
```
Comment (j0g3sc) [inline docs/administration.platforms.md:39]:
```suggestion
The Platform Control Plane is the central entry point for performing any tasks on your meshPlatform or insights you want to gather on it.
```

File: docs/administration.platforms.md [hunk 15]
```diff
@@ -1,37 +1,70 @@
...
+
+### Settings
 
-## Restrict Platform Access
+This tab is mainly responsible for managing how your meshPlatform is shown and used inside meshStack. You can provide general information
+about your Platform like a display name or the Web Console Url meshStack will link to when the user wants to actually access the cloud platform.
 
-Access to a specific platform can be restricted in the Administration Area via "Platforms" -> "Platform Instances". This feature is helpful
+You can also group your meshPlatforms into meshLocations to e.g. combine multiple private cloud platforms that are running in the same data center
+into one meshLocation. Or you are using e.g. multiple Azure Tenants for your different stages and want to group them into an Azure meshLocation.
```
Comment (j0g3sc) [inline docs/administration.platforms.md:53]: > Or you are using e.g. multiple Azure Tenants for [..]

In general I would avoid sentences that start with and / or

File: docs/administration.platforms.md [hunk 16]
```diff
@@ -1,37 +1,70 @@
...
+
+Additionally you can provide some Provider information of your meshLocation. You can describe how you operate your Cloud Platforms that belong to a meshLocation,
```
Comment (j0g3sc) [inline docs/administration.platforms.md:55]: I found `provider information` and `Provider information`, please check again that capitalization is the same every time.

File: docs/administration.platforms.md [hunk 17]
```diff
@@ -1,37 +1,70 @@
...
+
+#### Restrict Platform Access
+
+Access to a specific platform can be restricted via the "Restrictions" tab. This feature is helpful
 when a new meshPlatform shall be integrated, but initially only be visible to a few customers for integration testing.
+Or you have a dedicated Cloud Platform that shall only be available to certain meshCustomers in general.
 
-To restrict the meshPlatform, go to the meshPlatform edit screen and search for the customers the platform shall be restricted to.
+To restrict the meshPlatform, search for the customers the platform shall be restricted to.
 Add these customers via the *+* button.
```
Comment (j0g3sc) [inline docs/administration.platforms.md:65]: maybe connect the two senctences with and?

> To restrict the meshPlatform, search for the customers the platform shall be restricted to and add these customers via the *+* button.

File: docs/administration.platforms.md (hunk 11 above)
Comment (j0g3sc) [inline docs/administration.platforms.md:82]: i'd use unavailable instead of "not available"

Comment (j0g3sc) [reply docs/administration.platforms.md:82]: I'm in favor of getting rid of it and just adapt the description when the behavior is changed,

Comment (j0g3sc) [inline docs/administration.platforms.md:82]: maybe "platform's tenants" is a bit clearer then "platform tenants" ?

**Review by j0g3sc — CHANGES_REQUESTED** [review]
you describe behavior quite well. :+1:
Mainly minor improvement remarks from my side in addition to Jelle's comments.
Please make sure to use consistent names + capitalization throughout the document.

File: docs/administration.platforms.md (hunk 1 above)
Comment (sttomm) [reply docs/administration.platforms.md:6]: added a sentence before

File: docs/administration.platforms.md (hunk 6 above)
Comment (sttomm) [reply docs/administration.platforms.md:30]: moved this to the end of the Platform Creation section as the restriction part is also covered in the settings tab

File: docs/administration.platforms.md [hunk 18]
```diff
@@ -1,37 +1,70 @@
...
+tenant creation, replication and metering of your Cloud Platforms. The maintenance includes
+general information about the Cloud Platform as well as structuring and configuring them.
 
-## Platform Notifications
+A central part of it also includes the application of security and governance which is covered in the
+[Landing Zone Administration](administration.landing-zones.md).
 
-Administrators can inform users about platform specific events (updates, new features, service disruptions, etc.) by creating platform
-notifications:
+Getting an insight into all tenants that are being used in a certain Cloud Platform can be found in the
```
[the remark this answers is not in the document]
Comment (sttomm) [reply docs/administration.platforms.md:13]: replaced it with jelle's proposed text

File: docs/administration.platforms.md (hunk 5 above)
Comment (sttomm) [reply docs/administration.platforms.md:29]: changed this sentence anyway to mainly link to the dedicated meshLocation section

**Review by Jelledb — APPROVED** [review]
Nice job! Very useful addition to our docs

**Review by j0g3sc — APPROVED** [review]
